tests/x11/yast2_lan_restart_devices.pm

```diff
@@ -79,37 +88,44 @@ sub add_device {
 
 sub select_special_device_tab {
     my $device = shift;
+
     open_network_settings;
     send_key 'tab';
     send_key 'tab';
     send_key 'home';
-    send_key_until_needlematch "yast2_lan_device_${device}_selected", 'down';
-    send_key 'alt-i';                     # Edit NIC
+    send_key_until_needlematch ["yast2_lan_device_${device}_selected", "yast2_lan_device_workaround-bsc1004643"], 'down', 5;
+    if (record_soft_failure_bsc1111483) {
```

Comment:
similar to above, how about `return if record_soft_failure_bsc1111483;` just to save some space? :)

Reply:
I had it like that at the begining, but Travis didn't pass, it is detected as duplicated code. Neither worked `return if (record_soft_failure_bsc1111483)` nor `return if (record_soft_failure_bsc1111483()).

Reply:
@okurz thanks for the tip in the call, no duplicate, it is the name of the function likely, which our make rule was identifying it wrongly. Let's see if it works changing it.